MultiplayerMod/Game/GameState.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HarmonyLib;
+using UnityEngine;
+
+namespace MultiplayerMod.Game;
+
+[HarmonyPatch]
+public static class GameState {
```

Comment:
Name seems to be miss aligned with patched class and the class purpose as well.
Maybe `GamePatch`, `GameOnPrefabInitPatch` or `LastSelectedObjectState` might be better.

Reply:
What are the concerns? 
It's current game state that happens to have the last selected object (and also contains some instructions on how to obtain it).

Reply:
Now it's more obvious, let's review in the future.